x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/terms/helpers.ts

```diff
@@ -208,7 +208,8 @@ export function getDisallowedTermsMessage(
 function checkLastValue(column: GenericIndexPatternColumn) {
   return (
     column.operationType !== 'last_value' ||
-    !(column as LastValueIndexPatternColumn).params.showArrayValues
+    (['number', 'date'].includes(column.dataType) &&
```

Comment:
Does it make sense to allow boolean type?

Reply:
No, I think it makes sense like this.